### DISCUSSION TITLE
Opposite transform of DivisiblePad?

### DISCUSSION THREAD

**Discussion by jmlipman** [Project-MONAI/tutorials#1591] [Q&A] [answered]
Hi,

I'm using DivisiblePad to pad images on the fly. It's very handy because I have variable-size images and all I care is that they can fit into the model. At test time I also have variable-size images, and I need to undo this operation, i.e., the model gets a padded image and the prediction needs to be unpadded (cropped)  to be compared with its ground truth. CenterSpatialCrop could be useful but it only allows for a fixed roi_size for every image. Is there any nice way to solve this?

Thanks!

**Comment by KumoLiu** [comment, marked as answer]
Hi @jmlipman, thanks for your interest here.

I think you mean `inverse` and `DivisiblePad` is invertible, you could refer to this [tutorial](https://github.com/Project-MONAI/tutorials/blob/main/modules/inverse_transforms_and_test_time_augmentations.ipynb).

Hope it helps, thanks!

**Reply by jmlipman** [reply]
I think this is what I was looking for. Thank you!
